### ISSUE TITLE
Cant get seed sync to start for first time

### ISSUE THREAD

**Issue by dthomas710** [ipsingh06/seedsync#77]
I'm a huge novice so bear with me. I'm trying to get SeedSync going on an unraid machine and I appear to have it set up and configured correctly, but I can't access my remote server and it's unclear why. The error I have at the top of the screen says "An error occurred while scanning the local system." Here are the logs from SS. 

 2020/11/29 16:41:16 - INFO - seedsync.ServerActionHandler - Received a restart action

2020/11/29 16:41:16 - INFO - seedsync - Exiting Seedsync

2020/11/29 16:41:16 - INFO - seedsync.ServerActionHandler - Received a restart action

2020/11/29 16:41:17 - ERROR - LocalScanner.LocalScanner - Caught SystemScannerError Traceback (most recent call last): File "/app/python/controller/scan/local_scanner.py", line 28, in scan result = self.__scanner.scan() File "/app/python/system/scanner.py", line 81, in scan raise SystemScannerError("Path does not exist: {}".format(self.path_to_scan)) system.scanner.SystemScannerError: Path does not exist: /mnt/user/Unsorted

2020/11/29 16:41:18 - ERROR - seedsync.ControllerJob - Caught exception in job ControllerJob Traceback (most recent call last):
  File "/app/python/common/job.py", line 44, in run
    self.execute()
  File "/app/python/controller/controller_job.py", line 29, in execute
    self.__controller.process()
  File "/app/python/controller/controller.py", line 199, in process
    self.__propagate_exceptions()
  File "/app/python/controller/controller.py", line 525, in __propagate_exceptions
    self.__local_scan_process.propagate_exception()
  File "/app/python/common/app_process.py", line 124, in propagate_exception
    raise exc.re_raise()
  File "/app/python/common/app_process.py", line 30, in re_raise
    raise self.ee.with_traceback(self.tb)
  File "/app/python/common/app_process.py", line 87, in run
    self.run_loop()
  File "/app/python/controller/scan/scanner_process.py", line 90, in run_loop
    files = self.__scanner.scan()
  File "/app/python/controller/scan/local_scanner.py", line 31, in scan
    raise ScannerError(Localization.Error.LOCAL_SERVER_SCAN, recoverable=False)
controller.scan.scanner_process.ScannerError: An error occurred while scanning the local system.

2020/11/29 16:41:18 - ERROR - seedsync - Caught exception Traceback (most recent call last):
  File "/app/python/seedsync.py", line 164, in run
    controller_job.propagate_exception()
  File "/app/python/common/job.py", line 77, in propagate_exception
    raise exc_info[1].with_traceback(exc_info[2])
  File "/app/python/common/job.py", line 44, in run
    self.execute()
  File "/app/python/controller/controller_job.py", line 29, in execute
    self.__controller.process()
  File "/app/python/controller/controller.py", line 199, in process
    self.__propagate_exceptions()
  File "/app/python/controller/controller.py", line 525, in __propagate_exceptions
    self.__local_scan_process.propagate_exception()
  File "/app/python/common/app_process.py", line 124, in propagate_exception
    raise exc.re_raise()
  File "/app/python/common/app_process.py", line 30, in re_raise
    raise self.ee.with_traceback(self.tb)
  File "/app/python/common/app_process.py", line 87, in run
    self.run_loop()
  File "/app/python/controller/scan/scanner_process.py", line 90, in run_loop
    files = self.__scanner.scan()
  File "/app/python/controller/scan/local_scanner.py", line 31, in scan
    raise ScannerError(Localization.Error.LOCAL_SERVER_SCAN, recoverable=False)
controller.scan.scanner_process.ScannerError: An error occurred while scanning the local system.

2020/11/29 16:41:20 - INFO - seedsync.Controller - Exited controller 

Please provide any assistance you can, thank you!

**Comment by ipsingh06** [comment]
This error in your log seems to indicate that the local directory is not readable by SeedSync:
```
Path does not exist: /mnt/user/Unsorted
```

Could you verify this directory exists and is accessible to Seedsync

**Comment by quadcom** [comment]
Don't forget Linux systems are case sensitive.

**Comment by dthomas710** [comment]
Incredibly fast response! I believe it is accessible to SS based on the mapping I did for my unraid server, but again I'm a beginner so I'm likely doing something wrong. 

Here is the path mapping I put in for the SS docker container on my unraid server. 

![image](https://user-images.githubusercontent.com/68824903/100572552-96b12700-328a-11eb-917b-56ca9f0c6656.png)

![image](https://user-images.githubusercontent.com/68824903/100572592-b1839b80-328a-11eb-985d-83d7f2550381.png)

**Comment by ipsingh06** [comment]
Thanks for the screenshots. You shouldn't be using the paths on Host side inside the Seedsync settings because those paths are not visible to the Seedsync app. Use the container path in Seedsync settings, i.e. `/downloads`.

**Comment by dthomas710** [comment]
Hey thank you! That got it working, I'm definitely a docker beginner so I appreciate you assisting me.

One thing I noticed once I got it working is that the only way to use this software is in a sync based functionality. I was hoping this would be more of a on-demand download based solution like filezilla or cuteFTP. Is there any way for me to use it in this way? If not, hopefully one day in the future it will be an option!

Thanks again for being willing to help a novice like myself. Issue closed.
